src/pages/iou/MoneyRequestModal.js

```diff
@@ -194,35 +194,15 @@ const MoneyRequestModal = (props) => {
      * @returns {String}
      */
     const titleForStep = useMemo(() => {
-        const isSendingMoney = props.iouType === CONST.IOU.MONEY_REQUEST_TYPE.SEND;
-        if (currentStepIndex === 1 || currentStepIndex === 2) {
-            const formattedAmount = props.numberFormat(
-                amount, {
-                    style: 'currency',
-                    currency: props.iou.selectedCurrencyCode,
-                },
-            );
-            if (isSendingMoney) {
-                return props.translate('iou.send', {
-                    amount: formattedAmount,
-                });
-            }
-            return props.translate(
-                props.hasMultipleParticipants ? 'iou.split' : 'iou.request', {
-                    amount: formattedAmount,
-                },
-            );
-        }
         if (currentStepIndex === 0) {
-            if (isSendingMoney) {
+            if (props.iouType === CONST.IOU.MONEY_REQUEST_TYPE.SEND) {
                 return props.translate('iou.sendMoney');
             }
             return props.translate(props.hasMultipleParticipants ? 'iou.splitBill' : 'iou.requestMoney');
         }
-
-        return props.translate(steps[currentStepIndex]) || '';
+        return props.translate('iou.cash');
         // eslint-disable-next-line react-hooks/exhaustive-deps -- props does not need to be a dependency as it will always exist
-    }, [amount, currentStepIndex, props.hasMultipleParticipants, props.iou.selectedCurrencyCode, props.iouType, props.numberFormat, steps]);
+    }, [currentStepIndex, props.translate]);
```

Comment:
Just noting that `props.hasMultipleParticipants` is still a dependency, but since this prop doesn't change throughout the component lifecycle we don't need to include it here.